### DISCUSSION TITLE
create 2 zigbee networks on a single server.

### DISCUSSION THREAD

**Discussion by sobrarbe** [Koenkk/zigbee2mqtt#17848] [Devices]
Hello, I wanted to know if there is the possibility of putting a sonoff USB Dongle coordinator on the same server as Home Assistant through zigbee2mqtt and another Lan coordinator with ZHA and thus create 2 zigbee networks on the same server as Home Assistant.  thank you.

**Comment by deviantintegral** [comment]
Duplicated at https://github.com/Koenkk/zigbee2mqtt/discussions/17850
